src/links/airlines/as.js

```diff
@@ -21,7 +21,7 @@ function printAS() {
       childMinAge: 2
     });
     if (!pax) {
-      printNotification("Error: Failed to validate Passengers in printAAc1");
+      printNotification("Error: Failed to validate Passengers in printAaSabre");
```

Comment:
I know this was already incorrect, but would you mind reverting or fixing it to show `AS` instead? Otherwise I could fix it separately

Reply:
Great catch.. I missed that my find all and replace grabbed it. Corrected to printAS.